### DISCUSSION TITLE
No space left on device

### DISCUSSION THREAD

**Discussion by rafazelaya** [photoprism/photoprism#3163] [Q&A]
After setting up photoprism in docker I ran the complete scan and crashed photoprism.  Here's the error.  

Error response from daemon: mkdir /var/lib/docker/overlay2/d81e12bfe6006478ede76d05fb46ee1696824eb5faa2327d91cf46544296df44-init: no space left on device. 

Any advice for a novice like myself?  

Thanks

**Comment by graciousgrey** [comment]
Did you find our troubleshooting guides already :) ? 
 
https://docs.photoprism.app/getting-started/troubleshooting/#fatal-server-errors

 https://docs.photoprism.app/getting-started/troubleshooting/docker/#disk-space

**Reply by rafazelaya** [reply]
> Did you find our troubleshooting guides already :) ?
> 
> https://docs.photoprism.app/getting-started/troubleshooting/#fatal-server-errors
> 
> https://docs.photoprism.app/getting-started/troubleshooting/docker/#disk-space

@graciousgrey I browsed through some.  These look specific to my error.  Thanks
